libcloud/storage/drivers/backblaze_b2.py

```diff
@@ -292,7 +316,7 @@ def download_object_as_stream(self, obj, chunk_size=None):
                                 success_status_code=httplib.OK)
 
     def upload_object(self, file_path, container, object_name, extra=None,
-                      verify_hash=True, headers=None):
+                      verify_hash=True, headers=None, iterator=None):
```

Comment:
@Kami I just noticed backblaze has a sneak peak of their new api method - b2_upload_part

Would it be better to implement [this](https://www.backblaze.com/b2/docs/b2_upload_part.html) as the upload_object_via_stream method?